gulpfile.js

```diff
@@ -1,8 +1,27 @@
 var gulp = require('gulp');
+var ts = require('gulp-typescript');
 var gutil = require('gulp-util');
+var sourcemaps = require('gulp-sourcemaps');
+var ddescribeIit = require('gulp-ddescribe-iit');
+var karmaServer = require('karma').Server;
 var webpack = require('webpack');
-var gulpFormat = require('gulp-clang-format');
+var del = require('del');
+var merge = require('merge2');
 var clangFormat = require('clang-format');
+var gulpFormat = require('gulp-clang-format');
+var runSequence = require('run-sequence');
+
+// Transpiling & Building
+
+var buildProject = ts.createProject('tsconfig.json', {declaration: true});
+
+gulp.task('clean:build', function() { return del('dist/'); });
+
+gulp.task('cjs', function() {
+  var tsResult = gulp.src(['src/**/*.ts', '!src/**/*.spec.ts']).pipe(ts(buildProject));
```

Comment:
We start to have `'src/**/*.ts'` and other globs duplicated in few places. Could we pull it to some kind of config object? Ex.: https://github.com/pkozlowski-opensource/ng2-play/blob/master/gulpfile.js#L3-L5